packages/cadl-ranch/src/scenarios-resolver.ts

```diff
@@ -1,7 +1,7 @@
 import { ScenarioMockApi } from "@azure-tools/cadl-ranch-api";
 import { Scenario } from "@azure-tools/cadl-ranch-expect";
 import { Operation } from "@typespec/compiler";
-import { join, relative, resolve } from "path";
+import { join, relative, resolve, dirname, sep } from "path";
```

Comment:
Same here you can revert those changes I believe after the other PR is merged

Reply:
updated.